src/injected/main-window-initializer.ts

```diff
@@ -206,9 +204,12 @@ export class MainWindowInitializer extends WindowInitializer {
         await Promise.all(asyncInitializationSteps);
     }
 
-    @autobind
     protected dispose(): void {
-        super.dispose();
+        // Using .bind(this) instead of an arrow function because ES5 only supports accessing methods (not properties) of super
+        // When we move to target ES6, this can be simplified to super.dispose()
```

Comment:
>/ When we move to target ES6, this can be simpl [](start = 9, length = 47)

can we just target to es6? chrome anyway supports es6 rite?

Reply:
I think this would be a good idea, but out of scope from this PR